builder/README.md

```diff
@@ -0,0 +1,73 @@
+## Instructions on how to use **Makefile.docker**
+
+In order to have a controlled building environment for tracee, tracee provides
+a [Makefile.docker](../Makefile.docker) file that allows you to create and use
+docker controlled environments to build & test `tracee-ebpf` and
+`tracee-rules`.
+
+### Creating a builder container
```

Comment:
Can we add one or two sentences explaining why we even have two builder containers? I know it has to do with musl libc and glibc, but for curious individuals it can be helpful.

Reply:
Definitely. Good idea.